src/appleseed/foundation/platform/system.cpp

```diff
@@ -421,6 +429,388 @@ uint64 System::get_process_virtual_memory_size()
     return static_cast<uint64>(rss) * sysconf(_SC_PAGESIZE);
 }
 
+// ------------------------------------------------------------------------------------------------
+// FreeBSD.
+// ------------------------------------------------------------------------------------------------
```

Comment:
Yes, the contents of a namespace is indented (with 4 spaces). The only exception is for top-level namespaces (`foundation`, `renderer`) since that would indent entire files but provide no practical value.

`TrivialX86Cache`, that's correct!